instrumentation/okhttp/okhttp-3.0/library/src/main/java/io/opentelemetry/instrumentation/okhttp/v3_0/OkHttpTelemetryBuilder.java

```diff
@@ -24,6 +26,7 @@ public final class OkHttpTelemetryBuilder {
       new ArrayList<>();
   private List<String> capturedRequestHeaders = emptyList();
   private List<String> capturedResponseHeaders = emptyList();
+  @Nullable private Set<String> knownMethods = null;
```

Comment:
I really dislike these `null`s here; but this was the least invasive way of implementing an optionally unset set of methods. I've an idea on how to refactor these, but I'll tackle that in a separate PR.